### PR TITLE
Enhancement/fix automatic code

### PR DESCRIPTION
**Pull request by trombomart**

### CODE REVIEW

File: src/test/java/nl/tudelft/thefirstorder/web/rest/CameraResourceIntTest.java [hunk 1]
```diff
@@ -93,6 +93,36 @@ public void initTest() {
 
     @Test
     @Transactional
+    public void createCameraWithId() throws Exception {
+        int databaseSizeBeforeCreate = cameraRepository.findAll().size();
+        camera.setId(123L);
```
Comment (cadw) [inline src/test/java/nl/tudelft/thefirstorder/web/rest/CameraResourceIntTest.java]: What is "123L", i.o.w. is this hardocoded/is there another way to do this?

File: src/test/java/nl/tudelft/thefirstorder/web/rest/CameraResourceIntTest.java [hunk 2]
```diff
@@ -93,6 +93,36 @@ public void initTest() {
...
+        // Create the Camera
+        restCameraMockMvc.perform(post("/api/cameras")
```
Comment (cadw) [inline src/test/java/nl/tudelft/thefirstorder/web/rest/CameraResourceIntTest.java]: Maybe it is a good idea to define this as a constant of some kind.

File: src/test/java/nl/tudelft/thefirstorder/web/rest/CameraResourceIntTest.java [hunk 3]
```diff
@@ -93,6 +93,36 @@ public void initTest() {
...
+        assertThat(cameras).hasSize(databaseSizeBeforeCreate);
+    }
+
+    @Test
+    @Transactional
+    public void updateCameraNoId() throws Exception {
+        int databaseSizeBeforeUpdate = cameraRepository.findAll().size();
+        // Update the Camera
+        Camera updatedCamera = new Camera();
+        restCameraMockMvc.perform(put("/api/cameras")
```
Comment (cadw) [inline src/test/java/nl/tudelft/thefirstorder/web/rest/CameraResourceIntTest.java]: Maybe it is a good idea to define this as a constant of some kind.

**Comment by cadw** [comment]
I have some small remarks about the code quality, however, it is a simple fix and if you look at these remarks I think it is mergeable.

File: src/test/java/nl/tudelft/thefirstorder/web/rest/CameraResourceIntTest.java (hunk 1 above)
Comment (trombomart) [inline src/test/java/nl/tudelft/thefirstorder/web/rest/CameraResourceIntTest.java]: @cadw this is a test so it doesn't matter which id you pick. So in this case this isn't really hardcoding.

File: src/test/java/nl/tudelft/thefirstorder/web/rest/CameraResourceIntTest.java (hunk 2 above)
Comment (trombomart) [inline src/test/java/nl/tudelft/thefirstorder/web/rest/CameraResourceIntTest.java]: @cadw constant is a good idea I will make a commit

**Comment by hungnguyen94** [comment]
`CameraResource.java` and `Camera.java` were also changed. They give a few checkstyle warnings. Could you look at that?

**Pull request merged**